main.py

```diff
@@ -156,7 +164,7 @@ def level3_keyboard(message, *args, **kwargs):
 
 
 @check_database
```

Comment:
https://github.com/acetone415/talkback_bot/pull/9/files#r551295108

Reply:
@denisovkv Check the last commit)